examples/Utils/ArgumentNames.php

```diff
@@ -75,8 +75,8 @@ final class ArgumentNames
     public const FREEFORM_KEYWORD_TEXT = 'freeformKeywordText';
     public const GCLID = 'gclid';
     public const GEO_TARGET_CONSTANT_ID = 'geoTargetConstantId';
-    public const GMB_ACCESS_TOKEN = 'gmbAccessToken';
-    public const GMB_EMAIL_ADDRESS = 'gmbEmailAddress';
+    public const BUSINESS_PROFILE_ACCESS_TOKEN = 'businessProfileAccessToken';
+    public const BUSINESS_PROFILE_EMAIL_ADDRESS = 'businessProfileEmailAddress';
```

Comment:
This should be `BUSINESS_PROFILE_EMAIL` (same in the code example `AddBusinessProfileLocationExtensions`)

Reply:
+1